build.sbt

```diff
@@ -181,6 +181,6 @@ lazy val dottySettings = List(
 )
 
 TaskKey[Unit]("dottyCompile") := {
-  compile.in(minitestJVM, Test).value
-  compile.in(lawsJVM, Compile).value // Test fail to compile
```

Comment:
Why is the comment removed? Are the test in `lawsJVM` compiling? If so, the line should be changed to:
```scala
compile.in(lawsJVM, Test).value
// or if the tests run as well:
test.in(lawsJVM, Test).value
```

Reply:
That currently fails with 
```
[error] undefined: new <notype> # 9166: TermRef(TypeRef(TermRef(ThisType(TypeRef(NoPrefix,module class reflect)),module annotation),EnableReflectiveInstantiation),<init>) at frontend
```
an unrelated issue.

Reply:
That is a scala.js annotation.

Reply:
Added comment again